x-pack/plugins/ml/public/ui_actions/apply_influencer_filters_action.tsx

```diff
@@ -67,14 +71,18 @@ export function createApplyInfluencerFiltersAction(
         })
       );
     },
-    async isCompatible({ embeddable, data }) {
+    async isCompatible({ embeddable, data, ...rest }) {
```

Comment:
nit, `rest` is unused

Reply:
Removed in https://github.com/elastic/kibana/pull/138994/commits/083c565b8e6274052aea7d7934f7396d7d6f57aa